src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx

```diff
@@ -52,6 +52,7 @@ export interface IAlterationEnrichmentContainerProps {
     patientLevelEnrichments: boolean;
     onSetPatientLevelEnrichments: (patientLevel: boolean) => void;
     comparisonStore?: ComparisonStore;
+    alterationsRowData: AlterationEnrichmentRow[];
```

Comment:
we probably don't need this additional property. we can get it from the `comparisonStore`